packages/graphql/src/tsconfig.production.json

```
@@ -2,7 +2,10 @@
     "extends": "../../../tsconfig.base.json",
     "compilerOptions": {
         "rootDir": ".",
-        "outDir": "../dist"
+        "outDir": "../dist",
+        "module": "CommonJS",
```

OCD casing suggestion: https://www.typescriptlang.org/docs/handbook/modules/theory.html#the-module-output-format

```suggestion
        "module": "commonjs",
```